pkg/config/config.go

```diff
@@ -394,6 +394,9 @@ func newConfig(buildConfig []byte, rewriteDockerRepositoryName bool) (*Config, e
 		return nil, errors.Errorf("Unknown outputkind. Only DockerImage and ImageStreamTag supported, was %s", outputKind)
 	}
 	logrus.Debugf("Pushing to %s/%s:%s", dockerSpec.OutputRegistry, dockerSpec.OutputRepository, dockerSpec.TagWith)
+
+	logrus.Debugf("Output registry: %s", dockerSpec.OutputRegistry)
```

Comment:
Denne dupliserer output som allerede finnes to linjer over.

Reply:
Fikset